packages/SwingSet/src/kernel/parseKernelSlots.js

```diff
@@ -11,17 +11,13 @@ import { assert, details as X } from '@agoric/assert';
 // clist for the recipient vat.
 
 /**
- * Parse a kernel slot reference string into a kernel slot object:
- *   {
- *      type: STRING, // 'object', 'device', or 'promise'
- *      id: Nat
- *   }
+ * Parse a kernel slot reference string into a kernel slot object: { type:
+ * STRING, // 'object', 'device', or 'promise' id: Nat }
```

Comment:
More unacceptable mangling.

Reply:
Same as above. Without a code block there's no way for tools to know this was intended to be JS.
```
 * Parse a kernel slot reference string into a kernel slot object:
 * ```js
 * {
 *   type: STRING, // 'object', 'device', or 'promise'
 *   id: Nat
 * 
 * }
 * ```
```

Reply:
Master renders like this in IDE:
<img width="544" alt="Screen Shot 2022-03-23 at 1 04 18 PM" src="https://user-images.githubusercontent.com/21505/159786295-a7110d1b-c941-4068-aac0-61766537b122.png">

Reply:
Agreed. I propose we do it incrementally.

1. Use this PR to determine our target (configuration options, best practices)
2. Introduce those options in a separate PR that uses `overrides` to opt-in one package to formatting (https://github.com/Agoric/agoric-sdk/pull/4906 )
3. For each package, a codeowner creates a PR that has a) opts their package into jsdoc formatting, b) manually fixes jsdoc that won't be understood by the formatter (i.e. not valid jsdoc) and c) runs `yarn format`
4. Once nearly all packages are opted in, remove the `overrides` so that it applies to the whole repo

If there is agreement to this I'll document in #4900 as a plan.

Reply:
Very cool! I agree.

Reply:
I nominate Zoe as the next package. If you'll (@turadg) make the initial pass, then @erights and I will review.